src/screens/Member/signin.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import {
+    View,
+    Text,
+    TextInput, TouchableOpacity, KeyboardAvoidingView, Platform
+} from "react-native";
+import styles from "./styles";
+import Icon from 'react-native-vector-icons/FontAwesome';
+import {NavigationScreenProp} from "react-navigation";
+
+
+interface IProps {
+    navigation: NavigationScreenProp<any, any>
+}
+
+export default class Signin extends React.PureComponent<IProps, any> {
+
+    constructor(props: any) {
+        super(props);
+
+        this.state = {
+            emailOrPhone: "",
+            password: "",
+            showPassword: false,
+            validate: null
```

Comment:
i think this property describe wether the form is valid or not. so can be more explicit like `isValid` or `valid`.